src/com/jdngray77/htmldesigner/backend/BackgroundTask.kt

```diff
@@ -1,25 +1,52 @@
 package com.jdngray77.htmldesigner.backend
 
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
 /**
  * Runs tasks in the background
  */
-object BackgroundTask {
+object BackgroundTask : Subscriber {
+
+    private val threadPool = Executors.newCachedThreadPool()
+
+    init {
+        EventNotifier.subscribe(this, EventType.USER_EXIT)
+    }
+
 
     fun runOnUIThread(runnable: Runnable) {
```

Comment:
Mark this depricated, as it's not populated.

Reply:
Marked as depriciated with TODO.

Reply:
Lol i meant just comment it with //TODO so it shows in the todo window, the replace with suggests to someone to use a different method instead